src/docs/product/issues/issue-details/performance-issues/consecutive-http/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Consecutive HTTP"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/consecutive-http/
+description: "Learn more about Consecutive HTTP and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Consecutive HTTP issues are created when a set of sequential HTTP spans consistently exceeds a duration threshold.
+
+## Detection Criteria
+
+The detector for this performance issue looks for a set of sequential, non-overlapping HTTP spans. It intentionally ignores non-HTTP spans when finding this set, as the browser commonly performs other asynchronous operations such as loading resources, between HTTP spans. 
+
+Once this sequence is found, the following must hold true for each HTTP span:
+
+- The HTTP method must be a GET, POST, DELETE, PUT, PATCH
+- The HTTP URL must not begin with _next/static/ or _next/data/
+
+Once these spans are found, the following must also hold true:
+
+- The duration of each HTTP span must exceed a threshold.
+- The time between each HTTP span must exceed a threshold.
+- The number of sequential HTTP spans must exceed a threshold.
+
+If Sentry doesn't detect a Consecutive HTTP issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
+
+## Span Evidence
+
+You can identify a Consecutive HTTP problem by four main aspects in the "Span Evidence" section:
+
+- Transaction name
+- Sequential HTTP Spans
+
+![Consecutive HTTP span evidence](span-evidence.png)
+
+You can view the span evidence by going to the "Issues" page in Sentry, selecting your project, selecting the Consecutive HTTP Queries error you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.
+
```

Comment:
```suggestion
You can find additional information about your Consecutive HTTP issue by looking at four main aspects in the "Span Evidence" section:

- **Transaction name**
- **Sequential HTTP Spans**

![Consecutive HTTP span evidence](span-evidence.png)

You can view the span evidence by going to the **Issues** page in Sentry, selecting your project, selecting the Consecutive HTTP Queries error you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.

```
One big question - is saying "Sequential" and "Consecutive" the same thing? (Seeing it called "Sequential HTTP Spans in the image of the UI). Would be great if we were consistent.

Reply:
Hmm, good question. I think we'd want to emphasize "consecutive", "sequential" is more about order but here we're more concerned about the fact that there are non-overlapping spans next to each other. I can make some changes to the wording here hopefully

Reply:
Actually, I've been reading how we use "sequential" a bit more in these docs and I think it might make sense to keep these as is. There usually can be work that happens in-between the spans so they're technically not "next" to each other, but in sequence they are.